tests/test_quick_start.py

```diff
@@ -46,42 +76,45 @@ def setup_module():
 
 
 def teardown_module():
-    test_configs_path = tests_dir / 'configs'
-    shutil.rmtree(str(test_configs_path))
+    shutil.rmtree(str(tests_dir / "configs"))
 
 
 def download(full=None):
-    cmd = ["python3", "-m", "deeppavlov.download"]
+    cmd = "python3 -m deeppavlov.download"
     if full:
-        cmd.append("-all")
-    sp.run(cmd)
+        cmd += " -all"
+    pexpect.run(cmd)
 
 
-@pytest.mark.parametrize("model", [k for k, v in MCQR.items()])
+@pytest.mark.parametrize("model,conf_file,model_dir", [(m, c, md) for m in PARAMS.keys() for c, md in PARAMS[m].keys()])
 class TestQuickStart(object):
 
     @staticmethod
-    def interact(config, query="exit"):
-        p = sp.Popen(["python3", "-m", "deeppavlov.deep", "interact", config],
-                     stdin=sp.PIPE, stdout=sp.PIPE, stderr=sp.PIPE)
-        out, _ = p.communicate(f"{query}".encode())
-        return out
-
-    def test_downloaded_model_exist(self, model):
-        if not tests_dir.parent.joinpath('download').exists():
+    def interact(conf_file, model_dir, qr_list=None):
+        qr_list = qr_list or []
+        p = pexpect.spawn("python3", ["-m", "deeppavlov.deep", "interact", str(conf_file)], timeout=None)
+        for (query, expected_response) in qr_list:  # works until the first failed query
+            p.expect(":: ")
+            p.sendline(query)
+            p.expect(">> ")
+            actual_response = p.readline().decode().strip()
+            assert expected_response == actual_response, f"Error in interacting with {model_dir} ({conf_file}): {query}"
```

Comment:
It would be more informative to also show expected and actual responses in the error message.
Also, is it possible to run all the queries and not stop after the first failure?